projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules-active.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Endpoint to manage plugin and theme update schedules active status.
+ *
+ * Example: https://public-api.wordpress.com/wpcom/v2/update-schedules/$ID/active
+ *
+ * @package automattic/scheduled-updates
+ */
+
+use Automattic\Jetpack\Scheduled_Updates;
+use Automattic\Jetpack\Scheduled_Updates_Active;
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_Update_Schedules_Active
+ */
+class WPCOM_REST_API_V2_Endpoint_Update_Schedules_Active extends WP_REST_Controller {
+	/**
+	 * The namespace of this controller's route.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * The base of this controller's route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'update-schedules';
+
+	/**
+	 * WPCOM_REST_API_V2_Endpoint_Update_Schedules_Active constructor.
+	 */
+	public function __construct() {
+		// Priority 9 to run before the main endpoint and avoid "/active" being treated as a schedule ID.
```

Comment:
Would it be treated as a schedule ID? Looking at the route, there should be one provided